python/langsmith/wrappers/_openai.py

```diff
@@ -160,12 +167,59 @@ def _reduce_completions(all_chunks: List[Completion]) -> dict:
     return d
 
 
+def _create_usage_metadata(oai_token_usage: dict) -> UsageMetadata:
+    input_tokens = oai_token_usage.get("prompt_tokens", 0)
+    output_tokens = oai_token_usage.get("completion_tokens", 0)
```

Comment:
would you ever get "None" as a value here? Maybe better to do "or 0" to avoid a None + None scenario?